File: src/main/java/com/example/airline/controller/HelloController.java
package com.example.airline.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/4444")
public class HelloController {

    @GetMapping
    public String hello() {
        return "Hello, world!11111";
    }
}

File: mainApp/src/main/java/com/example/airline/controller/HelloController.kt
package com.example.airline.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HelloController {

    @GetMapping("/")
//    @Suppress("FunctionOnlyReturningConstant")
    fun hello(): String {
        return "Hello, world!"
    }
}